fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/AggregateIndexExpansionVisitor.java

```diff
@@ -243,15 +244,20 @@ private Quantifier constructGroupBy(@Nonnull final Quantifier selectWhereQun, @N
                 .map(Column::getValue)
                 .collect(ImmutableList.toImmutableList());
         final var selectQunValue = selectWhereQun.getRangesOver().get().getResultValue();
-        final var aliasMap = AliasMap.identitiesFor(Sets.union(selectQunValue.getCorrelatedTo(), groupingValues.stream().flatMap(v -> v.getCorrelatedTo().stream()).collect(ImmutableSet.toImmutableSet())));
-        final var pulledUpGroupingValuesMap = selectQunValue.pullUp(groupingValues, aliasMap, ImmutableSet.of(), selectWhereQun.getAlias());
-        final var pulledUpGroupingValues = groupingValues.stream().map(groupingValue -> {
-            if (!pulledUpGroupingValuesMap.containsKey(groupingValue)) {
-                throw new RecordCoreException("could not pull grouping value " + groupingValue)
-                        .addLogInfo(LogMessageKeys.VALUE, groupingValue);
-            }
-            return pulledUpGroupingValuesMap.get(groupingValue);
-        }).collect(ImmutableList.toImmutableList());
+        final var aliasMap = AliasMap.identitiesFor(
+                Sets.union(selectQunValue.getCorrelatedTo(),
+                        groupingValues.stream().flatMap(v -> v.getCorrelatedTo().stream())
+                                .collect(ImmutableSet.toImmutableSet())));
+        final var pulledUpGroupingValuesMap =
+                selectQunValue.pullUp(groupingValues, aliasMap, ImmutableSet.of(), selectWhereQun.getAlias());
+        final var pulledUpGroupingValues =
+                groupingValues.stream().map(groupingValue -> {
+                    if (!pulledUpGroupingValuesMap.containsKey(groupingValue)) {
+                        throw new RecordCoreException("could not pull grouping value " + groupingValue)
+                                .addLogInfo(LogMessageKeys.VALUE, groupingValue);
+                    }
+                    return pulledUpGroupingValuesMap.get(groupingValue);
+                }).collect(ImmutableList.toImmutableList());
```

Comment:
These are all white space/line break changes, right? Just making sure I'm not missing anything